README.md

```diff
@@ -46,3 +47,12 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+
+## Noah Mulvaney and I are working together on this exercise.
+
+### Worse Case Time Complexity
+Based on the theoretical analysis of tsp held karp we are looking at every possible permutation of the cities for the possible starts. For a given start we are finding the minimum of the possible $(|V|-1)!$ permutations of the $|V|-1$ other cities. There are $|V|!$ permutations of the cities for all starts. Our implementation has a time complexity of $\Theta(|V|!)$.
```

Comment:
...but you're also memoizing on the start city, right? What effect does that have on complexity?

Reply:
Are you sure that there is no repeated work that is avoided in this way?

Reply:
We have gone through our `helper` recursive function and we are storing data from every call. In our If statements that checks if we have stored a value for that pair of arguments, we have put a `console.log` to verify if that return is every used, and we do not get any output. We are pretty certain that there are no calls of our `helper` function with the same pair of arguments i.e. there is no repeated work.

Reply:
Hmm, as far as I can tell your code is correct, but there should be repeated work that you're avoiding with the memoization. It might help to step through an example of ~4 cities to see where the repeated work happens without memoization.